### PR TITLE
Merge of duckduckgo/community-platform-static into duckduckgo/community-platform

### PR DESCRIPTION
**Pull request by jagtalon**
It looks like it works, but needs more testing and we also have to remove instances of `DDGC::Static`. It's currently pulling in the SCSS files that we need in https://github.com/duckduckgo/community-platform/blob/1ceeacb5949eedce7f2b7430dc76cfe87987ca4f/src/scss/content/main.scss and we're compiling it as `content.scss`.

So currently `ddgc.css`, `ia.css`, and `content.css` is being built, but we can easily merge them into just one file in the future.

For @MariagraziaAlastra and @sdougbrown.

### CODE REVIEW

**Comment by jagtalon** [comment]
@jbarrett could you check it out as well? Is it possible to deploy it in view without releasing or maybe only releasing locally?

File: src/scss/content/_global.scss [hunk 1]
```diff
@@ -0,0 +1,989 @@
...
+				background: -moz-linear-gradient(top,  #ffffff 0%, #f0f0f0 100%);
+				background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#ffffff), color-stop(100%,#f0f0f0));
+				background: -webkit-linear-gradient(top,  #ffffff 0%,#f0f0f0 100%);
+				background: -o-linear-gradient(top,  #ffffff 0%,#f0f0f0 100%);
+				background: -ms-linear-gradient(top,  #ffffff 0%,#f0f0f0 100%);
+				background: linear-gradient(to bottom,  #ffffff 0%,#f0f0f0 100%);
+				-pie-background: linear-gradient(to bottom,  #ffffff 0%,#f0f0f0 100%);
```
Comment (russellholt) [inline src/scss/content/_global.scss]: what the heck

**Comment by jbarrett** [comment]
@jagtalon :+1: I am away this afternoon for the weekend. Cool if I evaluate on Monday?

You can do a `dzil build` and deploy the tarball to view with:

`script/ddgc_release.sh view DDGC-1.037.tar.gz`

(As long as your key is installed for ddgc@view)

...though deployments take place on Friday, which means a lot of stuff goes on in view so check out the community-platform room first

**Comment by MariagraziaAlastra** [comment]
@jbarrett Monday is perfect, I didn't want to roll this out today anyway. 

@jagtalon Thanks, that's a lot of work! I'm delaying merging this just to make sure everything is alright :smile:

**Comment by jbarrett** [comment]
One thing to check is 404s in firebug's network output and fonts load properly in new forum post and feedback pages :)

**Comment by jagtalon** [comment]
@jbarrett @MariagraziaAlastra thanks! Yes I'm definitely cool with that--I don't wanna break the site. I've been checking the console and haven't found anything yet.

File: script/ddgc_dev_server.psgi [hunk 2]
```diff
@@ -22,7 +22,6 @@ builder {
 
     mount '/' => DDGC::Web->new->psgi_app;
 
-    mount "/ddgc_static" => Plack::App::File->new( root => DDGC::Static::sharedir )->to_app;
```
Comment (jbarrett) [inline script/ddgc_dev_server.psgi]: We should also lose the `use DDGC::Static` above

**Comment by jbarrett** [comment]
So, I ran `grunt` to generate static files and `script/ddgc_dev_server.sh` to kick off a new dev instance. Is this correct?

I am getting 404s for the files ddg-feedback.ttf and ddg-feedback.woff, meaning there are no icons on feedback pages:

missing feedback icons

I also get no icons on the front page or site buttons (though I don't think they use the feedback font):

missing frontpage icons

Thanks!

**Comment by jagtalon** [comment]
@jbarrett off, alright i'll check it out!

**Comment by jbarrett** [comment]
@jagtalon Thanks. Let me know what help I can provide.

File: script/ddgc_dev_server.psgi (hunk 2 above)
Comment (jagtalon) [inline script/ddgc_dev_server.psgi]: @jbarrett thanks good find!

**Comment by jagtalon** [comment]
@jbarrett @MariagraziaAlastra fonts should be working now. Will try to deploy on a staging server.

**Comment by jbarrett** [comment]
:+1: Staging LGTM, thanks @jagtalon!

**Comment by MariagraziaAlastra** [comment]
Thanks @jagtalon! :cactus:

**Pull request merged**